src/patients/related-persons/RelatedPersonTab.tsx

```diff
@@ -21,6 +21,7 @@ const RelatedPersonTab = (props: Props) => {
   const { permissions } = useSelector((state: RootState) => state.user)
   const [showNewRelatedPersonModal, setShowRelatedPersonModal] = useState<boolean>(false)
   const [relatedPersons, setRelatedPersons] = useState<Patient[] | undefined>(undefined)
+  const [dataFetched, setDataFetched] = useState<boolean>(false)
```

Comment:
I think we should use `isLoading` here to stay consistent with other parts of the application. Thoughts?

Reply:
Will Do.